src/domain/user/user.service.ts

```diff
@@ -53,6 +55,19 @@ export class UserService {
     return this.userRepository.findOne({ email: email });
   }
 
+  async userExistsAadCt(email: string): Promise<boolean> {
```

Comment:
Looks like this function is not used - keep it or now? Removing it also removes knowledge of MSGraph from teh User service which is probably correct.

Reply:
Addressed in commit c590131c0265b53e16dcf1b2160f43fc4613f547